src/index.js

```diff
@@ -1549,14 +1561,24 @@ const _startRenderLoop = () => {
     }
 
     // wait for next frame
-    immediate = setImmediate(_renderLoop);
+    if (_isVisible()) {
+      loop = setImmediate(_renderLoop);
+      cancelLoop = clearImmediate;
+    } else {
+      loop = setTimeout(_renderLoop, 100);
```

Comment:
This looks like a good design, but this part worries me. From my understanding, this means that there is a new `100ms` delay before we enter a VR render loop.

Also, we are adding more test processing to the render loop here that I don't think is really needed.

Can we instead perhaps start/stop the render loop when there are windows to render, rather than checking and delaying each frame?